docs/Installation/Installation_Guide.rst

```diff
@@ -1,75 +1,67 @@
-.. spelling::
-
-   Quickstart
-   subdirectory
-   Todo
-
-
 Installation Guide
-++++++++++++++++++
+==================
 
-While it is possible to run Zenko on a single machine, it's designed for
-cluster operation. If you can set up a Kubernetes cluster on your own, review
-the **General Cluster Requirements** and skip to **Installing Zenko**, below.
+While it is possible to run Zenko on a single machine, it’s designed for
+cluster operation. If you can set up a Kubernetes cluster on your own,
+review the :ref:`General Cluster Requirements` and skip to
+:ref:`Installing Your Zenko Instance`.
 
-Otherwise, you can set up a cluster quickly using MetalK8s_, Scality's
-open-source Kubernetes cluster project, as described in "Setting Up a Metal K8s
-Cluster," below.
+Otherwise, you can set up a cluster quickly using
+`MetalK8s <https://github.com/scality/metal-k8s/>`__, Scality’s
```

Comment:
While the old url will redirect for now it's best to switch all references of `metal-k8s` to `metalk8s` to prevent any issues in the future.

Reply:
I agree. This may have been an older change coming in off a conflict resolution. As it stood last time I looked in on this, the unhyphenated URL was not resolving. Will address looking forward.

Reply:
URL is fixed. Instances of the metalk8s venv still show the hyphen, so that remains in the documentation.

Reply:
I don't know how to pull content review apart from "all these assets." I edited the docs while converting to RST. Sugar is cast upon the white sands. 

These changes have also been superseded, in part, by ZENKO-559. If you must be thorough, the best I can recommend is to limit scope to RST docs and leave the rest to tox. For this JIRA, the only significant changes will have been to docs/Installation/Installation.rst. The rest is nit-picking.